docs-gb/kubernetes/hpa-rps-autoscaling.md

```diff
@@ -246,29 +276,111 @@ spec:
         averageValue: 3
 ```
 
-In the two HPA manifests above, the scaling metric is exactly the same, and uses the exact same parameters: this is to ensure that both the Models and the Servers are scaled up/down at approximately the same time. Small variations in the scale-up time are expected because each HPA samples the metrics independently, at regular intervals. If a Model gets scaled up slightly before its corresponding Server, the model is currently marked with the condition ModelReady "Status: False" with a "ScheduleFailed" message until new Server replicas become available. However, the existing replicas of that model remain available and will continue to serve inference load.
+In the two HPA manifests above, the scaling metric is exactly the same, and uses the exact same
+parameters: this is to ensure that both the Models and the Servers are scaled up/down at
+approximately the same time. Small variations in the scale-up time are expected because each HPA
+samples the metrics independently, at regular intervals.
+
+{% hint style="info" %}
+If a Model gets scaled up slightly before its corresponding Server, the model is currently
+marked with the condition ModelReady "Status: False" with a "ScheduleFailed" message until new
+Server replicas become available. However, the existing replicas of that model remain available
+and will continue to serve inference load.
+{% endhint %}
+
+In order to ensure similar scaling behaviour between Models and Servers, the number of
+`minReplicas` and `maxReplicas`, as well as any other configured scaling policies should be kept
+in sync across the HPA for the model and the server.
+
+#### Details on custom metrics of type Object
+
+The HPA manifests above use metrics of type "Object" that fetch the data used in scaling
+decisions by querying k8s metrics associated with a particular k8s object.  The endpoints that
+HPA uses for fetching those metrics are the same ones that we tested in the previous section
+using `kubectl get --raw ...`. Because we have configured the Prometheus Adapter to expose those
```

Comment:
```suggestion
using `kubectl get --raw ...`. Because you have configured the Prometheus Adapter to expose those
```
https://developers.google.com/style/person